packages/core/test/lib/resourceShape.test.ts

```diff
@@ -0,0 +1,137 @@
+import rdf from '@zazuko/env-node'
+import { expect } from 'chai'
+import defaultResourceShapeLookup from '../../lib/resourceShape.js'
+import type { KopflosConfig } from '../../lib/Kopflos.js'
+import Kopflos from '../../lib/Kopflos.js'
+import inMemoryClients from '../support/in-memory-clients.js'
+import { ex } from '../support/ns.js'
+import { createStore } from '../support/testData.js'
+
+describe('lib/resourceShape', () => {
+  let options: KopflosConfig
+
+  beforeEach(createStore(import.meta.url, { format: 'trig' }))
```

Comment:
Loads RDF from `resourceShape.test.ts.trig` and sets mocha `graph` and `store` properties to the mocha context.

The contents of the store/graph will be taken from a graph constructed from the test `describe/context` titles